### ISSUE TITLE
Code to dump cluster centers as PDB or XTC

### ISSUE THREAD

**Issue by sbhakat** [msmbuilder/msmbuilder2022#27]
Wondering is there convenient function to dump cluster centers as either .pdb or .xtc? This will be super helpful.

**Comment by bojunliu0818** [comment]
Hi, I’m not sure there’s a current function for that (probably not), but I’ll look into it. I’ll let you know if I there is one or if a solution becomes available in the future!

**Comment by sbhakat** [comment]
@bojunliu0818 This is a key functionality.

**Comment by bojunliu0818** [comment]
Yes, thanks for raising this problem. As far as I know, currently there is no convenient function in msmbuilder to do this directly, and people may need to do this manually with mdtraj or some other tools in md software. I will track this issue and consider including this functionality in the future release of msmbuilder.
